### ISSUE TITLE
Clicking on "show more references" returns user to top of list

### ISSUE THREAD

**Issue by jesseallhands** [Couchers-org/couchers#4397]
When reading references on someone's profile, clicking on the "show more references" button takes you back to the top of the references list (both on desktop and mobile). This is annoying if the person has a lot of references and you want to read through a lot of them because every time you have to scroll down to the bottom of the list again. 

When clicking on "show more references", it should not return the user to the top of the list. It should keep them in the same spot in the list.

**Comment by vas2525** [comment]
@jesseallhands Not sure how it worked before, but it seems fine now. Take a look when you get a chance.

Env: Mac + Chrome

https://github.com/user-attachments/assets/c99687a3-9382-4a0f-9685-109040ee2b0f

**Comment by jesseallhands** [comment]
It's either a mobile-only or firefox-only but cause I was able to replicate it just now on mobile Firefox

**Comment by vas2525** [comment]
You're right, iOS 18.6 + Chrome/Safari also reproducible, but on Mac 15.7.1 + Chrome is working fine

Also, reproducible on Mac 15.7.1 + Safari
